libs/openFrameworks/gl/ofCubeMap.cpp

```diff
@@ -383,30 +383,33 @@ bool ofCubeMap::load( ofCubeMapSettings aSettings ) {
 			// figure out the number of mip maps //
 			data->maxMipLevels = log2(data->settings.preFilterRes) + 1;
 			
-			std::string encFolder = data->settings.cacheDirectory;
+			of::filesystem::path encFolder = data->settings.cacheDirectory;
```

Comment:
these should prob be in another PR but fine for this one. 

Maybe we should start using `auto` instead of `of::filesystem::path` for local vars going forward as it will make the code more adaptable in the future.

Reply:
I think this specific one should not be auto, because data->settings.cacheDirectory is a string, so I'm building an of::filesystem::path from that.